docs/sources/installation/debian.md

```diff
@@ -28,9 +28,10 @@ To install the latest Debian package (may not be the latest Docker release):
 
 To verify that everything has worked as expected:
 
-    $ sudo docker run -i -t ubuntu /bin/bash
+    $ sudo docker run --rm hello-world
 
-Which should download the `ubuntu` image, and then start `bash` in a container.
+which should download the `hello-world` image, and then print a friendly Hello
```

Comment:
This command downloads and runs the `hello-world` image in a container. When the container runs, it prints an informational message about your Docker installation. Then, it exits.